vtrace/__init__.py

```diff
@@ -708,6 +713,20 @@ def removeBreakpoint(self, id):
         # Remove cached breakpoint code
         Breakpoint.bpcodeobj.pop(id, None)
 
+    def _updateBreakAddresses(self):
```

Comment:
whenever we have a LOAD_LIBRARY event, we want to look through deferred breakpoints and see if we can resolve them.

Reply:
sure is.  i'll consolidate.